### PR TITLE
SISRP-21404 - Emergency Contacts - unable to edit a migrated contact

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-21404

Fixes default values before edit instead of on save.

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js [hunk 1]
```diff
@@ -10,8 +10,9 @@ angular.module('calcentral.controllers').controller('EmergencyContactController'
 
   angular.extend($scope, {
     currentObject: {},
+    defaultCountry: 'USA',
     emptyObject: {
-      country: 'USA',
+      country: this.defaultCountry,
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js]: Nice. I've learned something.

Comment (dfkaye) [inline src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js]: :)

File: src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js [hunk 2]
```diff
@@ -70,36 +71,35 @@ angular.module('calcentral.controllers').controller('EmergencyContactController'
...
+      geoCode: item.geoCode,
+      houseType: item.houseType,
+      inCityLimit: item.inCityLimit,
+      num1: item.num1,
+      num2: item.num2,
+      phone: phone.phone,
+      phoneType: phone.phoneType,
+      extension: phone.extension,
+      postal: item.postal,
+      state: item.state
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js]: @dfkaye I see you've moved "sanitization" to `showEdit`. Can the data become de-sanitized during edit and thus sent to CS as such?

Comment (dfkaye) [inline src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js]: @johncrossman don't think so - I tried it out - seemed OK

**Comment by johncrossman** [comment]
Thanks, @dfkaye  Mergalicious.

**Pull request merged**